diff --git a/config/desktop/focal/environments/mate/debian/postinst b/config/desktop/focal/environments/mate/debian/postinst
index aff07abb6..09e3d21b5 100755
--- a/config/desktop/focal/environments/mate/debian/postinst
+++ b/config/desktop/focal/environments/mate/debian/postinst
@@ -15,10 +15,10 @@ install -Dv /dev/null $profile
 # deconf dump org/nemo/ > nemo_backup
 # deconf dump org/mate/ > cinnamon_desktop_backup
 
-echo "[org/nemo/list-view]
+echo "[org/caja/list-view]
 default-visible-columns=['name', 'size', 'type', 'date_modified', 'owner', 'permissions']
 
-[org/nemo/preferences]
+[org/caja/preferences]
 quick-renames-with-pause-in-between=true
 show-advanced-permissions=true
 show-compact-view-icon-toolbar=false
@@ -31,7 +31,7 @@ show-list-view-icon-toolbar=false
 show-new-folder-icon-toolbar=true
 show-open-in-terminal-toolbar=true
 
-[org/nemo/window-state]
+[org/caja/window-state]
 geometry='800x550+550+244'
 maximized=false
 sidebar-bookmark-breakpoint=5
@@ -59,7 +59,7 @@ secondary-click-time=1.2
 
 [org/mate/desktop/background]
 picture-options='zoom'
-picture-uri='file:///usr/share/backgrounds/armbian/armbian03-Dre0x-Minum-dark-3840x2160.jpg'
+picture-filename='/usr/share/backgrounds/armbian/armbian03-Dre0x-Minum-dark-3840x2160.jpg'
 primary-color='#456789'
 secondary-color='#FFFFFF'
 
@@ -80,6 +80,9 @@ toolkit-accessibility=false
 [org/mate/desktop/media-handling]
 autorun-never=false
 
+[org/mate/desktop/peripherals/mouse]
+cursor-theme='default'
+
 [org/mate/desktop/screensaver]
 picture-options='zoom'
 picture-uri='file:///usr/share/backgrounds/armbian-lightdm/armbian03-Dre0x-Minum-dark-3840x2160'
@@ -90,6 +93,10 @@ secondary-color='#FFFFFF'
 num-workspaces=2
 theme='Numix'
 
+[org/mate/marco/general]
+num-workspaces=2
+theme='Numix'
+
 [org/mate/settings-daemon/peripherals/touchpad]
 disable-while-typing=true
 horiz-scroll-enabled=false
