diff --git a/config/desktop/jammy/environments/mate/config_base/packages b/config/desktop/jammy/environments/mate/config_base/packages
index 0227724a4..9f6718fe5 100644
--- a/config/desktop/jammy/environments/mate/config_base/packages
+++ b/config/desktop/jammy/environments/mate/config_base/packages
@@ -1,5 +1,19 @@
+adcli
+alsa-base
+alsa-utils
 anacron
 apport-gtk
+apturl
+at-spi2-core
+atril
+ayatana-indicator-application
+ayatana-indicator-datetime
+ayatana-indicator-notifications
+ayatana-indicator-printers
+ayatana-indicator-session
+ayatana-indicator-sound
+ayatana-settings
+bc
 blueman
 bluez
 bluez-cups
@@ -8,6 +22,14 @@ brltty
 brltty-x11
 cifs-utils
 colord
+ca-certificates
+caja
+caja-admin
+caja-open-terminal
+caja-rename
+caja-sendto
+caja-wallpaper
+cryptsetup
 cups
 cups-bsd
 cups-client
@@ -35,12 +57,14 @@ fonts-stix
 fonts-symbola
 fonts-ubuntu-console
 foomatic-db-compressed-ppds
+fortune-mod
 gdebi
 ghostscript-x
 gist
 gnome-orca
 gnome-disk-utility
 gnome-screenshot
+gnome-session-canberra
 gnome-user-docs-de
 gnome-user-docs-es
 gnome-user-docs-it
@@ -48,15 +72,15 @@ gnome-user-docs-pt
 gnome-user-docs-ru
 gnome-user-docs-sl
 gnome-system-monitor
-gstreamer1.0-packagekit
-gstreamer1.0-plugins-base-apps
+gstreamer1.0-alsa
+gstreamer1.0-plugins-base
+gstreamer1.0-plugins-good
 gstreamer1.0-pulseaudio
 gtk2-engines
 gtk2-engines-murrine
 gtk2-engines-pixbuf
 gvfs-backends
 hplip
-indicator-printers
 inputattach
 inxi
 kerneloops
@@ -69,8 +93,13 @@ language-pack-gnome-it
 language-pack-gnome-pt
 language-pack-gnome-ru
 language-pack-gnome-sl
+language-selector-common
+language-selector-gnome
 laptop-detect
 libatk-adaptor
+libcanberra-gtk-module
+libcanberra-gtk3-module
+libcanberra-pulse
 libcvc0
 libfont-afm-perl
 libfontconfig1
@@ -80,9 +109,11 @@ libgail-common
 libgl1-mesa-dri
 libgsettings-qt1
 libgtk2.0-bin
+libmtp-runtime
 libnotify-bin
 libproxy1-plugin-gsettings
 libproxy1-plugin-networkmanager
+libsasl2-modules
 libu2f-udev
 libwmf0.2-7-gtk
 libxapp1
@@ -90,6 +121,7 @@ libxcursor1
 lightdm
 lightdm-settings
 lxtask
+marco
 mate-accessibility-profiles
 mate-applet-appmenu
 mate-applet-brisk-menu
@@ -152,6 +184,7 @@ mate-window-menu-applet
 mate-window-title-applet
 mesa-utils
 mousepad
+mozo
 mousetweaks
 network-manager-config-connectivity-ubuntu
 network-manager-openvpn
@@ -171,6 +204,7 @@ printer-driver-all
 profile-sync-daemon
 pulseaudio-module-bluetooth
 redshift
+rfkill
 samba
 slick-greeter
 smbclient
@@ -185,6 +219,7 @@ tracker-miner-fs
 tree
 ubuntu-drivers-common
 update-inetd
+unzip
 update-manager
 update-manager-core
 update-notifier
@@ -195,6 +230,8 @@ wbrazilian
 wbritish
 wfrench
 witalian
+wireless-tools
+wpasupplicant
 wportuguese
 wspanish
 wswiss
@@ -209,10 +246,13 @@ xapps-common
 xarchiver
 xdg-dbus-proxy
 xdg-desktop-portal
+xdg-desktop-portal-gtk
 xdg-user-dirs
 xdg-user-dirs-gtk
 xfonts-base
 xinput
+xkb-data
+xorg
 xorg-docs-core
 xscreensaver
 xsensors
@@ -224,3 +264,6 @@ yelp
 yelp-xsl
 zenity
 zenity-common
+plank
+gnome-menus
+pluma
